setup.py

```diff
@@ -24,19 +24,19 @@
     long_description = f.read()
 
 install_requires = [
-    'jupyterhub',
-    'tornado',
-    'graphene-tornado'
+    'jupyterhub==1.0.*',
+    'tornado==6.0.*',
+    'graphene-tornado>=2.0.3,<2.1.0'
```

Comment:
Wouldn't that be `>=2.0.3,<=2.1.0`?

Reply:
I thought our current code was not ready for 2.1.0 yet, and that maybe you could bump it up to 2.1.* or so in your branch

Reply:
Yes, could do. I think it might work as is in this branch though, because it was just my subclassing of the handler that was the issue.

Reply:
Let's bump up to 2.1.* here then, and either merge your branch or cherry-pick the changes and merge it earlier if necessary 🚀

Reply:
👍